docs/go/getting-started.md

```diff
@@ -81,25 +81,25 @@ This file declares the `greet.v1` Protobuf package, a service called
 response structures. These package, service, and method names will reappear
 soon in our HTTP API's URLs.
 
-## Generate code
+## Generate code using Buf v2
```

Comment:
I don't think it is necessary to reference the config version in the header because it is self evident in the config file itself. (if we did want to reference it in the header, the correct way would be to call it "buf.gen.yaml v2", but like I said, I don't think this is necessary for the header)

```suggestion
## Generate code
```

Reply:
Absolutely. I have fixed it with [a2b3f3f](https://github.com/connectrpc/connectrpc.com/pull/162/commits/a2b3f3fd68f9107ceb7ac17f89e5cdddf564859e)